pkg/ccl/multiregionccl/regional_by_row_system_database_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Licensed as a CockroachDB Enterprise file under the Cockroach Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     https://github.com/cockroachdb/cockroach/blob/master/licenses/CCL.txt
+
+package multiregionccl
+
+import (
+	"context"
+	"testing"
+
+	"github.com/cockroachdb/cockroach/pkg/base"
+	"github.com/cockroachdb/cockroach/pkg/ccl/multiregionccl/multiregionccltestutils"
+	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
+	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
+	"github.com/cockroachdb/cockroach/pkg/util/log"
+)
+
+// This is a rather contrived test to demonstrate that one can successfully,
+// albeit with some arm-twisting, convince the sql layer to make regional-
+// by-row tables in the system database.
```

Comment:
Shouldn't this PR include a new CCL logic test config in which the system database is multi-region? Seems like it'd be worth doing because it'd be nice to add more test cases, then again I have no idea what kind of effort it involves. You be the judge.